util/test_importers.py

```diff
@@ -0,0 +1,54 @@
+import unittest
+
+from .importers import validate_label_eng, get_validate_label
+
+class TestValidateLabelEng(unittest.TestCase):
+
+    def test_numbers(self):
+        label = validate_label_eng("this is a 1 2 3 test")
+        self.assertEqual(label, None)
+
+class TestGetValidateLabel(unittest.TestCase):
+
+    def test_no_validate_label_locale(self):
+        class Args:
+            def __init__(self):
+                pass
+        args = Args()
```

Comment:
The type returned by `argparse` is public, it's `argparse.Namespace`. If you use it directly here instead of mocking it, you don't need the `.__dict__` dance in `get_validate_label`.

For an empty object:
`args = Namespace()`

For a parameter:
`args = Namespace(validate_label_locale='foo')`

Reply:
Ah, nice. I admit I was too lazy for this.